src/overlays/actors/ovl_En_Zf/z_en_zf.c

```diff
@@ -452,9 +452,15 @@ s16 EnZf_FindNextPlatformAwayFromPlayer(Vec3f* pos, s16 curPlatform, s16 arg2, G
         }
     }
 
-    // These functions have no side effects, so these two calls do nothing
+    //! @bug `altNextPlatform` can be -1 in certain conditions and cause an out of bounds access.
+    //! Under normal conditions, this doesn't cause problems because the data before `sPlatformPositions`
+    //! is section padding between .text and .data, so 0 gets read as a float.
+    //! Other compilers can shift things and result in a crash. Defining `AVOID_UB` will remove these function calls
+    //! entirely as they don't have any effect (and return values are not used).
+#ifndef AVOID_UB
```

Comment:
as in the previous comment,
```suggestion
    //! Other compilers can shift things and result in a crash.
#ifndef AVOID_UB
    // Defining `AVOID_UB` will remove these function calls entirely as they don't have any effect (and return values are not used).
```
(may need a line break)